src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADD_BOOK = 'ADD_BOOK';
```

Comment:
- [ ] It would be better to follow the ducks pattern in creating our action types. For example; `const ADD_BOOK 
= 'bookstore/books/ADD_BOOK'` and same for our Remove Book actions. Kindly do that for all action types